exo-sync/features/steps/steps.ls

```diff
@@ -21,7 +21,7 @@ module.exports = ->
     fs.write-file-sync path.join(@app-dir, filename), file-content
 
 
-  @Given /^The origin of "([^"]*)" contains a new commit not yet present in the local clone$/, (service-role) ->
+  @Given /^The origin of "([^"]*)" contains a new commit not yet present in the local clone$/, (service-type) ->
```

Comment:
An error had slipped past from one of the other PRs. This one instance of service-role needed to change to service-type.